scheduler/distributed_task_test.go

```diff
@@ -238,24 +245,22 @@ func TestDistributedSubscriptions(t *testing.T) {
 			localMockManager := &subscriptionManager{Fail: false}
 			schTask.RemoteManagers.Add("", localMockManager)
 			// Start task. We expect it to fail while subscribing deps
-			terrs := sch.StartTask(t.ID())
+			terrs := s.StartTask(t.ID())
 			So(terrs, ShouldNotBeNil)
 			Convey("So dependencies should have been unsubscribed", func() {
 				// Ensure that unsubscribe call count is equal to subscribe call count
 				// i.e that every subscribe call was followed by an unsubscribe since
 				// we errored
 				So(remoteMockManager.UnsubscribeCallCount, ShouldEqual, remoteMockManager.SubscribeCallCount)
-				So(localMockManager.UnsubscribeCallCount, ShouldEqual, localMockManager.UnsubscribeCallCount)
```

Comment:
**To reviewers**:
Changed
`So(localMockManager.UnsubscribeCallCount, ShouldEqual, localMockManager.UnsubscribeCallCount)`
(what does not have sense because it's always true)
to 
`So(localMockManager.UnsubscribeCallCount, ShouldEqual, localMockManager.SubscribeCallCount)`